src/Administration/Resources/app/administration/src/module/sw-product/view/sw-product-detail-variants/index.js

```diff
@@ -136,11 +136,24 @@ Component.register('sw-product-detail-variants', {
                     const groupCriteria = new Criteria();
                     groupCriteria
                         .setLimit(100)
-                        .setPage(1);
+                        .setPage(1)
+                        .setTotalCountMode(1);
```

Comment:
Can you use the ENUM? https://github.com/shopware/platform/blob/trunk/src/Administration/Resources/app/administration/src/core/data/criteria.data.ts#L8

Reply:
Sorry, I just noticed that there are already tickets and apparently also pull requests to fix this. How did I not notice this before? I'm going to close this PR now.